tests/python_client/milvus_client/test_milvus_client_insert.py

```diff
@@ -1342,5 +1344,768 @@ def test_milvus_client_insert_after_json_path_index(self, enable_dynamic_field,
+        self.create_collection(client, collection_name, default_dim, schema=schema, 
+                               consistency_level="Strong", index_params=index_params)
+        
+        # step 2: partial upsert a new field
+        row = cf.gen_row_data_by_schema(nb=default_nb, schema=schema)
+        self.upsert(client, collection_name, row, partial_update=True)
+
+        new_row = [{default_primary_key_field_name: i, default_int32_field_name: 99} for i in range(default_nb)]
+        error = {ct.err_code: 1, 
+                ct.err_msg: f"Attempt to insert an unexpected field `{default_int32_field_name}` to collection without enabling dynamic field"}
+        self.upsert(client, collection_name, new_row, partial_update=True, check_task=CheckTasks.err_res, check_items=error)
+        
+        self.drop_collection(client, collection_name)
+    
+    @pytest.mark.tags(CaseLabel.L1)
+    @pytest.mark.skip(reason="Partial update not ready")
+    def test_milvus_client_partial_update_mixed_data(self):
+        """
+        target:  Test PU will return error when provided mixed valid and invalid data
+        method:
+            1. Create a collection
+            2. partial upsert mixed valid and invalid data
+        expected: Step 2 should result fail
+        """
+        client = self._client()
+        schema = self.create_schema(client, enable_dynamic_field=False)[0]
+        schema.add_field(default_primary_key_field_name, DataType.INT64, is_primary=True, auto_id=False)
+        schema.add_field(default_vector_field_name, DataType.FLOAT_VECTOR, dim=default_dim)
+        schema.add_field(default_int32_field_name, DataType.INT32, nullable=True)
+        index_params = self.prepare_index_params(client)[0] 
+        index_params.add_index(default_primary_key_field_name, index_type="AUTOINDEX")
+        index_params.add_index(default_vector_field_name, index_type="AUTOINDEX")
+        index_params.add_index(default_int32_field_name, index_type="AUTOINDEX")    
+        collection_name = cf.gen_collection_name_by_testcase_name(module_index=1)
+        self.create_collection(client, collection_name, default_dim, schema=schema, 
+                               consistency_level="Strong", index_params=index_params)
+        
+        # step 2: partial upsert mixed valid and invalid data
+        rows = cf.gen_row_data_by_schema(nb=default_nb, schema=schema)
+        self.upsert(client, collection_name, rows, partial_update=True)
+        new_value = np.int32(99)
+        new_rows = [{default_primary_key_field_name: i if i % 2 == 0 else i*10,
+                     default_int32_field_name: new_value} for i in range(default_nb)]
+        
+        error = {ct.err_code: 1100, 
+                 ct.err_msg: f"fieldSchema({default_vector_field_name}) has no corresponding fieldData pass in: invalid parameter"}
+        self.upsert(client, collection_name, new_rows, partial_update=True,
```

Comment:
I think it is dup to test_milvus_client_partial_update_new_pk_with_missing_field